firmware/esp32/gdoor/src/gdoor_data.cpp

```diff
@@ -21,18 +21,18 @@
 
 // Map the HW Type field between bus value and human readable string
 std::map<int, const char*>GDOOR_DATA_HWTYPE = {
-    { 0xA1, "INDOOR_AUDIO"},
-    { 0xA0, "OUTDOOR_AUDIO"},
-    { 0xA3, "CONTROLLER_AUDIO"}
+    { 0xA1, "INDOOR"},
```

Comment:
FYI @DaSchaef, it seems that there is no differenciation between audio and video devices